spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
+RSpec.describe 'The Foods Index Page' do 
```

Comment:
You may not need this if you are testing for the attributes like you did in the `search_spec`